device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/IotHubTransport.java

```diff
@@ -1488,7 +1488,7 @@ private void sendPacket(IotHubTransportPacket packet)
                 }
             }
 
-            log.info("Sending message ({})", message);
+            log.debug("Sending message ({})", message);
```

Comment:
This could overwhelm logs while handling with a huge amount of messages, so move logging level from INFO to DEBUG